cli/standalone.rs

```diff
@@ -30,28 +36,41 @@ const MAGIC_TRAILER: &[u8; 8] = b"d3n0l4nd";
 /// This function will try to run this binary as a standalone binary
 /// produced by `deno compile`. It determines if this is a stanalone
 /// binary by checking for the magic trailer string `D3N0` at EOF-12.
-/// After the magic trailer is a u64 pointer to the start of the JS
-/// file embedded in the binary. This file is read, and run. If no
-/// magic trailer is present, this function exits with Ok(()).
+/// The magic trailer is followed by:
+/// - a u64 pointer to the JS bundle embedded in the binary
+/// - the u64 length of the bundle
+/// - a u64 pointer to JSON metadata (serialized flags) embedded in the binary
+/// - the u64 length of the metadata
```

Comment:
I don't think this is needed.

Reply:
I thought the redundant values would make the extraction more readable, but it's not that different. Switching to only use the pointers.